### DISCUSSION TITLE
Possible to migrate from lukesampson/scoop?

### DISCUSSION THREAD

**Discussion by nonethewizer** [Ash258/scoop#190] [Q&A] [answered]
Perhaps I've just missed it somewhere, but are the instructions somewhere for migrating from scoop? Been using scoop for awhile and hoping to avoid reinstalling everything from scratch.

Thanks!

Tim

**Comment by Ash258** [comment, marked as answer]
https://github.com/Ash258/Scoop-Core#installation
